apps/web/src/views/universalFarms/components/PoolListItemAction.tsx

```diff
@@ -32,8 +32,14 @@ const StyledConnectWalletButton = styled(ConnectWalletButton)`
 
 export const PoolListItemAction = memo(({ pool }: { pool: PoolInfo }) => {
   const { theme } = useTheme()
+
+  const handleClick = useCallback((e: React.MouseEvent) => {
```

Comment:
why should add a click event as it did not do anything here.

Reply:
the item is clickable with link, so the action should be stop bubble to prevent redirect to item detail page